CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ Entries in this file are grouped into several categories:
 ## [Unreleased]
 
 * Add virtio-blk support ([#234](https://github.com/opiproject/opi-intel-bridge/pull/234)).
```

Comment:
This will be later under section "Added" so we can reformulate:
* Add virtio-blk ... -> virtio-blk ...
* ...
* Set number ... -> Setting number of ...

Also, while still in progress we can add sth like:
* Redis support for ...

Reply:
> This will be later under section "Added" so we can reformulate:

Done

> Also, while still in progress we can add sth like

If we want to add smth in progress, I think it should be added under its own section